### ISSUE TITLE
SimpleIcons disappear from Lovelace UI and Sidebar.

### ISSUE THREAD

**Issue by FeromonDE** [vigonotion/hass-simpleicons#35]
Can you please check this issue:
https://github.com/home-assistant/android/issues/835

**Comment by vigonotion** [comment]
Can you try installing the Font Awesome custom component and check if the icons also disappear? I can't reproduce this on my device yet

**Comment by FeromonDE** [comment]
Yes, the icons from Font Awesome custom component also disappear, but only if i use my local network with Home Assistant Android App. On 4G with SSL-enabled url there is no problem with icons. Also there is no problem with icons in Mozilla.

**Comment by ozmule** [comment]
I also have the same issue. Icons randomly disappear on lovelace. They do however display on the mobile app on Android (2.3.3-full). 

![image](https://user-images.githubusercontent.com/63888591/92039622-b387ac80-edb8-11ea-9854-812419e2733d.png)

![image](https://user-images.githubusercontent.com/63888591/92039642-bedad800-edb8-11ea-8c0d-549c1f31d5b3.png)

**Comment by tomlut** [comment]
I'm seeing the simpleicon I assigned to a side menu item disappear on the iOS app but on the desktop (Firefox) it is always visible.

![IMG_0641](https://user-images.githubusercontent.com/10679300/93242655-94b5ec80-f7a0-11ea-8eaf-d0b9a53d6310.jpeg)

**Comment by vigonotion** [comment]
@tomlut @ozmule Could you both also check if you have the same problems with the Font Awesome integration? And also check if the problem only occurs on mobile data / wifi (or internal/external address)

**Comment by PepperPhil** [comment]
For me it also accours in my chrome browser with simpleicons integration. Mainly with panel_iframe I added in the config yaml. MDI icons are no problem.

![image](https://user-images.githubusercontent.com/69075397/93890557-c07a2a80-fcea-11ea-8308-4578caa09cf1.png)

![image](https://user-images.githubusercontent.com/69075397/93890803-046d2f80-fceb-11ea-9a33-d325590edd26.png)

**Comment by ozmule** [comment]
> @tomlut @ozmule Could you both also check if you have the same problems with the Font Awesome integration? And also check if the problem only occurs on mobile data / wifi (or internal/external address)

@vigonotion I don't have Font Awesome integration. The issues are via wifi on local (internal) address on Chrome and Android app

**Comment by nepozs** [comment]
I've noticed the same problem, HA versions 0.105.x 0.106.x simpleicons v. 1.8.0
Firefox 81.0.1 x64 win, no scriptblocking nor adblocking browser plugins.
It is remote (only) location connected by VPN.
I don't have Font Awesome integration. 

![Screenshot_2020-10-12 Lustro - Home Assistant_OK](https://user-images.githubusercontent.com/56661567/95792243-5bab6200-0ce3-11eb-9762-1e5607c22322.png)
![Screenshot_2020-10-12 Lustro - Home Assistant](https://user-images.githubusercontent.com/56661567/95792248-5e0dbc00-0ce3-11eb-943d-9ad6b63cae2e.png)

**Temporary solution** (works till  HA reboot/update) 
**Configuration menu->Integrations->Simple Icons->System Options for Simple Icons->UPDATE**


I've prepared test setup with very simple configuration and it looks like problem with some stage of plugin initialization, because problem exist only after HA update or HA system reboot; whereas HA restart probably is not causing the problem - it is difficult to judge repeatability - see at this (after HA reboot):
![Screenshot_2020-10-12 Lustro - Home Assistant_Partial_working](https://user-images.githubusercontent.com/56661567/95793461-ed1bd380-0ce5-11eb-8861-7f5547bd03d3.png)
view icon is not loaded, but entity icon is OK

**Comment by Blankf** [comment]
i have the same issue, with using English language.

running: 2020.12.2

Firefox worked, but Edge and Chrome both didnt.
maybe its chromium browser related?

**Comment by tompijls** [comment]
Confirming for me it works with safari (14.0.2) but not on chrome ( 87.0.4280.88) on mac os catalina 10.15.7 

running: 2020.12.2 in docker

**Comment by vigonotion** [comment]
I think I found the issue. The file is too big, it's probably somehow timing related. For a quick fix I now try to reduce the file size, and then when https://github.com/vigonotion/hass-simpleicons/issues/24 is implemented this shouldn't be an issue anymore

**Comment by IIAIronWolf** [comment]
Hoping to see this work soon! I just installed this for the first time and ran into this issue as well.

**Comment by nepozs** [comment]
@vigonotion 
I don't remember when I've had to use workaround described here:
https://github.com/vigonotion/hass-simpleicons/issues/35#issuecomment-707364641
so I think this issue can be closed.

**Comment by IIAIronWolf** [comment]
> @vigonotion 
> 
> I don't remember when I've had to use workaround described here:
> 
> https://github.com/vigonotion/hass-simpleicons/issues/35#issuecomment-707364641
> 
> so I think this issue can be closed.

Because you (one person) haven't had the issue, the issue should be closed?

**Comment by dcassese** [comment]
I have this same issue.  HA 2021.4.3    My SI icons dont show when on local network.  Oddly they show fine through Nabu Casa.

**Comment by drinfernoo** [comment]
I agree that this one should be closed.

**Comment by vigonotion** [comment]
@IIAIronWolf @tomlut @dcassese @FeromonDE 

Can you please test the v2 beta and add a comment if it works for you here: https://github.com/vigonotion/hass-simpleicons/discussions/194 ? I think my new implementation should fix this issue

**Comment by umrath** [comment]
I have the same issue when casting to a google device: si-icons do not show up

**Comment by vigonotion** [comment]
Please try v2 and open a new issue if neccessary:
Please try version 2 and open another issue id the issue still persists: https://github.com/vigonotion/hass-simpleicons/releases/tag/v2.0.0

**Comment by mspace00** [comment]
I am on 2.0.0 and can confirm icons not showing, however is appears only on Firefox. Icons display on android app and Chrome.

**Comment by Arcane1980** [comment]
This is STILL an issue in 2024.

I have been to multiple threads, multiple sites, looking for the solution. None of the solutions are permanent, and some don't even work at all.

I have Home Assistant Green, latest OS update, latest Simple Icons integration, done all of the cache tips and tricks.

I still can't see ANY of the Simple Icons on either mobile version of the app (S23 Ultra) or on a tablet (Android).

The Icons all appear with no issues on Chrome/Avast Browser.

**Comment by IIAIronWolf** [comment]
Not an issue for me anymore. HA OS on RPi 3 64-bit version with latest (as of this comment) version of HA available. Hasn't been an issue for many many versions now for me.
